ofnetAgent.go

```diff
@@ -452,12 +455,6 @@ func (self *OfnetAgent) EndpointAdd(epreg *OfnetEndpoint, ret *bool) error {
 		if !epreg.Timestamp.After(oldEp.Timestamp) {
 			return nil
 		}
-
-		// Uninstall the old endpoint from datapath
-		err := self.datapath.RemoveEndpoint(oldEp)
-		if err != nil {
-			log.Errorf("Error deleting old endpoint: {%+v}. Err: %v", oldEp, err)
-		}
 	}
```

Comment:
I don't think you meant to make this change. Merge problem?
Its required in case we receive endpoint updates